cmd/tiller-proxy/internal/handler/apprepos_handler.go

```diff
@@ -76,7 +76,7 @@ type appRepositoryRequest struct {
 
 type appRepositoryRequestDetails struct {
 	Name               string                 `json:"name"`
-	RepoURL            string                 `json:"repoUrl"`
+	RepoURL            string                 `json:"repoURL"`
```

Comment:
This issue was found during real-life testing: used wrong capitalization on the json tag which meant the `repoURL` sent from the dashboard was being ignored. Now it's consistent with `customCA` for example.